mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1937,6 +1937,28 @@ func ResourceContainerCluster() *schema.Resource {
 					},
 				},
 			},
+			"fleet": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Computed:    true,
+				MaxItems:    1,
+				Description: `Fleet information of the cluster.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"project": {
+							Type:         schema.TypeString,
+							Optional:         true,
```

Comment:
optional + computed means that it will send what is in the user's config if present, and otherwise will send whatever was last returned by the API. (Generally we use it for fields where the API returns a value even though none was sent, since otherwise users will always see a diff when they try to do any plan after the first apply.)

Does that sound correct for this field? (and the parent field?)

Reply:
The `optional` + `computed` behavior sounds right for this field.